setup_env.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
 export ZETH=`pwd`
+export ZETH_API_DIR=$ZETH/api
```

Comment:
ZETH_API_DIR and ZETH_KEYSTORE seem to longer be used.

Reply:
Alright. I didn't want to remove any env variables since I didn't want to cause any trouble on the python client side. I can remove them if they are not used anymore though, thanks